functions/lib/pubsub/processUploads.js

```diff
@@ -152,3 +158,40 @@ function compressImage(tempFilePath, imageManipulation) {
     return imageManipulation.compress(tempFilePath, 960);
   }
 }
+
+function readImageData(mediaPath, localFS) {
+  return function() {
+    const imageData = {}
+    return localFS.readFile(mediaPath)
+      .then((data) => {imageData.mediaData = data})
```

Comment:
It works because you're reading in raw data, if it were encoded differently then there could be a different number of characters vs bytes (e.g. with UTF-16). But you can verify this by looking at the buffer length vs the length reported for the same file by `ls -l`